docs/production/upgrade-or-modify.md

```diff
@@ -415,8 +415,7 @@ To upgrade the version of PostgreSQL on the Zulip server:
 2. Stop the server and take a backup:
 
     ```
-    sudo -i # Or otherwise get a root shell
-    supervisorctl stop all
+    /home/zulip/deployments/current/scripts/stop-server
```

Comment:
Is the thinking that we'll backport this script to 3.x?  Otherwise, we may want to leave this as-is.

Reply:
This one seems a little iffier, since the first step is "upgrade to the most recent zulip."  I'd be in favor of removing the parenthetical so we can have to-to-date instructions.

At very least, the `sudo -i` is at least incorrect here, since `supervisorctl` and `/home/zulip/deployments/current/manage.py backup` probably want to run as `zulip`.

Reply:
Hmm, yeah.